postgres_base_block.py

```diff
@@ -78,26 +77,22 @@ def configure(self, context):
         self.connect()
 
         # create list of column names for insertion validation
-        self._cur.execute("SELECT column_name FROM information_schema.columns \
-                          WHERE table_name = '{}';".format(self.table_name))
```

Comment:
It wouldn't have no, that was my bad. I completely missed it when reviewing the PR for the column validation.